openshift/prometheus-flatliner-live-deployment-template.yaml

```diff
@@ -42,6 +42,9 @@ parameters:
 - name: FLT_INFLUX_DB_DSN
   description: The URL to connect to an influx database
   required: true
+- name: FLT_LIVE_METRIC_DELAY
+  description: The URL to connect to an influx database
+  required: true
```

Comment:
maybe this can be optional? and default to `now`

Reply:
Should be fixed in the next commit, thanks.